lockbox-ios/Common/Resources/Constants.swift

```diff
@@ -9,7 +9,7 @@ import UIKit
 struct Constant {
     struct app {
         static let redirectURI = "https://mozilla-lockbox.github.io/fxa/ios-redirect.html"
-        static let faqURL = "https://lockbox.firefox.com/faq/"
+        static let faqURL = "https://lockbox.firefox.com/faq.html"
```

Comment:
To update at https://github.com/mozilla-lockbox/mozilla-lockbox.github.io/issues/46